api_tests/src/comment.spec.ts

```diff
@@ -42,6 +44,7 @@ beforeAll(async () => {
   await followBeta(gamma);
   let betaCommunity = (await resolveBetaCommunity(alpha)).community;
   if (betaCommunity) {
+    // Note: throughout this code, postRes shall be an alpha copy of the post
     postRes = await createPost(alpha, betaCommunity.community.id);
```

Comment:
Better to rename the var instead of putting a comment which can easily get outdated.